domains/Accounts/Tests/Feature/AccountsProfileTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Domains\Accounts\Tests\Feature;
+
+use Domains\Accounts\Database\Factories\UserFactory;
+use Domains\Accounts\Models\User;
+use Illuminate\Http\Response;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class AccountsProfileTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    protected User $user;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->user = UserFactory::new()->create();
+    }
+
+    /** @test */
+    public function guest_cannot_see_profile(): void
+    {
+        $response = $this->get(route('accounts.me'), ['Authorization' => 'Bearer ']);
+
+        $response->assertResponseStatus(Response::HTTP_UNAUTHORIZED);
+    }
+
+    /** @test */
+    public function authenticated_user_can_see_profile(): void
+    {
+        $token = auth()->login($this->user);
+        $this->get(route('accounts.me'), ['Authorization' => "Bearer {$token}"])
```

Comment:
We don't need to really generate a token. We can use the `actingAs()` method to mock an authenticated user (this also removes the need to inject the authorization header) and chain the rest of the methods.

Reply:
Ok, changed!